javascript/core/strings-ii/most-efficient-way-to-build-html-strings.md

```diff
@@ -1,23 +1,10 @@
 ---
 author: pawel
 
-levels:
-  - basic
-
 type: normal
 
 category: best practice
 
-standards:
-  javascript.evaluate-expressions.3: 10
-  javascript.write-expressions.1: 10
-
-tags:
-  - strings
-  - html
-  - introduction
-  - workout
-
 links:
   - '[jsperf.com](http://jsperf.com/string-concat-vs-array-join-10000){website}'
```

Comment:
I think we can remove this. when I tried it the link didn't work
![image](https://user-images.githubusercontent.com/19802557/77772706-7fa59f80-7040-11ea-8abe-e42c7ed1198e.png)

```suggestion
```